modules/serverless-ossm-verifying-installation-prerequisites.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+[id="serverless-ossm-verifying-installation-prerequisites_{context}"]
+= Verifying installation prerequisites
+
+Before installing and configuring the {SMProductShortName} integration with {ServerlessProductShortName}, verify that the prerequisites have been met.
+
+.Procedure
+
+. Check for conflicting gateways:
++
+.Example command
+[source,terminal]
+----
+$ oc get gateway -A -o jsonpath='{range .items[*]}{@.metadata.namespace}{"/"}{@.metadata.name}{" "}{@.spec.servers}{"\n"}{end}' | column -t
+----
++
+.Example output
+[source,terminal]
+----
+knative-serving/knative-ingress-gateway  [{"hosts":["*"],"port":{"name":"https","number":443,"protocol":"HTTPS"},"tls":{"credentialName":"wildcard-certs","mode":"SIMPLE"}}]
+knative-serving/knative-local-gateway    [{"hosts":["*"],"port":{"name":"http","number":8081,"protocol":"HTTP"}}]
+----
++
+This command should not return a `Gateway` that binds `port: 443` and `hosts: ["*"]`, except the `Gateways` in `knative-serving` and `Gateways` that are part of another {SMProductShortName} instance.
++
+[NOTE]
+====
+The mesh that {ServerlessProductShortName} is part of has to be distinct and preferably reserved only for {ServerlessProductShortName} workloads. That is because additional configuration, such as `Gateways`, might interfere with the {ServerlessProductShortName} gateways `knative-local-gateway` and `knative-ingress-gateway`. {SMProductName} only allows one Gateway to claim a wildcard host binding (`hosts: ["*"]`) on the same port (`port: 443`). If another Gateway is already binding this configuration, a separate mesh has to be created for {ServerlessProductShortName} workloads. 
```

Comment:
```suggestion
The mesh that {ServerlessProductShortName} is part of must be distinct and preferably reserved only for {ServerlessProductShortName} workloads. That is because additional configuration, such as `Gateways`, might interfere with the {ServerlessProductShortName} gateways `knative-local-gateway` and `knative-ingress-gateway`. {SMProductName} only allows one Gateway to claim a wildcard host binding (`hosts: ["*"]`) on the same port (`port: 443`). If another Gateway is already binding this configuration, a separate mesh has to be created for {ServerlessProductShortName} workloads. 
```